agent_build/common.py

```diff
@@ -0,0 +1,341 @@
+# Copyright 2014-2021 Scalyr Inc.
```

Comment:
Do we already run lint and other checks (black, etc.) on this new directory / package? If not, let's please update lint / tox targets to do that.